public/apps/configuration/panels/permission-list/permission-list.tsx

```diff
@@ -305,7 +306,11 @@ export function PermissionList(props: AppDependencies) {
             fetchData();
             addToast({
               id: 'saveSucceeded',
-              title: getSuccessToastMessage('Action group', action, groupName),
+              title: `${getSuccessToastMessage(
+                'Action group',
+                action,
+                groupName
+              )} ${getClusterInfoIfEnabled(dataSourceEnabled, dataSource)}`,
```

Comment:
do you think this should be combined under `getSuccessToastMessage` which accepts dataSource as additional parameter?

Reply:
This is a good suggestion - let me take a look at the usages of this and make sure it makes sense for all of them, if it does I will make the change.